independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/handlers/ProjectInfoCommandHandler.java

```diff
@@ -38,14 +43,14 @@ public QuarkusCommandOutcome execute(QuarkusCommandInvocation invocation) throws
         final boolean logStatePerModule = invocation.getValue(ProjectInfo.PER_MODULE, false);
```

Comment:
Haven't you removed the per-module option in this PR?

Reply:
Added deprecated, it was unused